tests/integration/api_v2/http/transactions.test.js

```diff
@@ -174,9 +174,9 @@ describe('Transactions API', () => {
 		});
 	});
 
-	describe('Retrieve transaction list by address', () => {
+	describe('Retrieve transaction list by sender/recipient address', () => {
```

Comment:
```suggestion
	describe('Retrieve transaction list by recipient address', () => {
```

Reply:
Given test suit is testing both (negative and positive) scenarios for both sender and recipient address.